packages/terra-core-docs/src/terra-dev-site/test/icon/IconAll.test.jsx

```diff
@@ -1,320 +1,321 @@
-import React from 'react';
-import classNames from 'classnames/bind';
-import Icon from 'terra-icon/lib/IconExports';
-import styles from './IconAll.module.scss';
+import React from "react";
+import classNames from "classnames/bind";
+import Icon from "terra-icon/lib/IconExports";
+import styles from "./IconAll.module.scss";
```

Comment:
Why are these being changed to a double quote? I think the linter would complain about using double quote instead of single.

Reply:
Prettier again. I'll give this a fix and shut Prettier off.